src/nxt_socket.h

```diff
@@ -92,15 +103,15 @@ typedef union {
 
 
 NXT_EXPORT nxt_socket_t nxt_socket_create(nxt_task_t *task, nxt_uint_t family,
```

Comment:
The same as `nxt_socket_bind()` below.

Reply:
Thanks, I got it, let's continue on this topic in this conversation: https://github.com/nginx/unit/pull/670#discussion_r851761720